internal/cloudapi/v2/handler.go

```diff
@@ -596,11 +596,11 @@ func (h *apiHandlers) GetComposeStatus(ctx echo.Context, id string) error {
 		if err != nil {
 			return HTTPError(ErrorMalformedOSBuildJobResult)
 		}
-		var buildJobResults []worker.OSBuildKojiJobResult
+		var buildJobResults []worker.OSBuildJobResult
```

Comment:
Could we get in trouble here if we call `GetComposeStatus` on an old job? Do we need some sort of translation?

Reply:
Yes, but we do not care. This is covered in the "Migration notes" part of the PR description.